src/main/resources/assets/thermoo/lang/en_us.json

```diff
@@ -4,7 +4,6 @@
     "attribute.thermoo.generic.frost_resistance": "Frost Resistance",
     "attribute.thermoo.generic.heat_resistance": "Heat Resistance",
     
-
     "commands.thermoo.temperature.exception.not_living_entity": "Target is not a living entity!",
```

Comment:
why was this line removed?

Reply:
Actually, it will cause confusion when you update the new version later and be mistakenly compared with other translations

Reply:
Right now I'm finishing up with scorchful, you can do the same. This is very confusing

Reply:
What are you confused about?

Reply:
Well, it's easy to understand because I compare the lines to see if you have updated the new version or made any changes. With a small number like 32-33 lines, it's simple. But with 300 words poured on it, it's very difficult

Reply:
Because it's in the pull section 🤣